functions/src/model/ciJobs.ts

```diff
@@ -138,8 +139,17 @@ export class CiJobs {
     repoVersionInfo: RepoVersionInfo,
     editorVersionInfo: EditorVersionInfo | null = null,
   ): CiJob => {
+    let status: JobStatus = 'deprecated';
+    if (
+      editorVersionInfo === null ||
```

Comment:
Logic looks good to me, but small question; When is `editorVersionInfo` `null`?.

Reply:
This will happen for base and hub images.